.github/workflows/build-publish-containers.yml

```diff
@@ -58,71 +57,23 @@ jobs:
       - name: Convert and store repository owner in lowercase
         run: |
           echo REPOSITORY_OWNER=$(echo ${{ github.repository_owner }} | tr '[:upper:]' '[:lower:]') >> $GITHUB_ENV
-          echo DOCKERFILE=${{ matrix.tag }}/Dockerfile >> $GITHUB_ENV
+          echo IMAGE_TAG=$(date +"%F")-${{ github.run_id }}.${{ github.run_number }} >> $GITHUB_ENV
 
       - name: Build and push to GitHub Packages
         uses: docker/build-push-action@v3
         with:
-          tags: ghcr.io/${{ env.REPOSITORY_OWNER }}/${{ matrix.tag }}-${{ matrix.platform }}
-          file: ${{ env.DOCKERFILE }}
-          platforms: linux/${{ matrix.platform }}
+          tags: |
+            ghcr.io/${{ env.REPOSITORY_OWNER }}/${{ matrix.tag }}:latest
```

Comment:
We should also keep the `${{ matrix.tag }}-amd64` and `${{ matrix.tag }}-arm64` tags, for backwards compatibility?

Reply:
Explicitly you mean? I'd say that the additional tag below (`${{ env.IMAGE_TAG }}`) will be enough for backward compatibility. The additional tag will get you the `linux/arm64` and 'linux/amd64` version as a manifest list.
I reckon that is enough.

Reply:
Well, no, what I would like to avoid is that people who are currently using `ghcr.io/easybuilders/rockylinux-8.7-amd64` start running into trouble - that should keep working (and pull in the latest revision of the image)

Reply:
Pity, if you guys would have started with only multi-arch the problem would not exists. Water under the bridge...
Lemme add another step (let's see how I get this going).
In case you want to add more architectures (`riscv`), please discourage the use of a separat repo for each arch.

Reply:
ok, that should do it.